src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs

```diff
@@ -22,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.SecureConnection.Cmdlet
     /// Returns the secure connection policy of a specific database.
     /// </summary>
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseSecureConnectionPolicy", SupportsShouldProcess = true), OutputType(typeof(DatabaseSecureConnectionPolicyModel))]
+    [Obsolete("Note that secure connection is deprecated and this command will be removed in a future release. Please use the SQL database blade in the Azure portal to view the connection strings", false)]
```

Comment:
close with PM.
I will point PS customers to the portal

Reply:
Do not use `Obsolete`. Use our breaking change attributes: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md

Reply:
done